integration/delete_test.go

```diff
@@ -66,7 +66,7 @@ func TestDeleteOrdered(t *testing.T) {
 		deletes            bson.A
 		ordered            bool
 		expectedRemovedIDs []string
-		expectedErr        string
+		// TODO: add expected error
```

Comment:
If you plan to do it in a separate issue, let's mention this issue here?

Reply:
I wanted to resolve it in the same issue, but different PR. Please let me know if you have any objections to this approach

Reply:
Yeah, that sounds reasonable!

I see codecov complains about some things, this might be something that needs to be fixed in this PR, or at least we need to describe test cases that would fix it (and we can implement test cases in a separate PR).

I'll take another look on Monday morning to give my final opinion.

(Please, don't work on weekends! I've just shown a really bad example of behavior with doing this review today, I'm sorry for that 🙈)